plugins/rbac-backend/src/role-manager/role-manager.ts

```diff
@@ -15,7 +14,7 @@ export class BackstageRoleManager implements RoleManager {
   private maxDepth?: number;
   constructor(
     private readonly catalogApi: CatalogApi,
-    private readonly log: Logger,
+    private readonly log: LoggerService,
```

Comment:
```suggestion
    private readonly logger: LoggerService,
```
We should change this to be uniform with other implementations.